combat-trainer.lic

```diff
@@ -721,6 +772,24 @@ class SpellProcess
     prepare_spell(data, game_state)
   end
 
+  def interrupt_casting(game_state)
+    fput('release spell') if game_state.casting
+    game_state.casting = false
+    game_state.cast_timer = nil
+  end
+
+  def check_consume(game_state)
+    return unless @is_necro
+    return if !game_state.prepare_consume
+    if game_state.casting && !game_state.casting_consume
```

Comment:
I could remove the entire bit about interruption, as I believe with the preserve ritual the body sticks around for a very very long time.